.github/workflows/actions.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
```

Comment:
Sorry can we avoid emojis in the text files?

Reply:
Sure thing